e2e/Microsoft.Azure.Devices.E2ETests/X509E2ETests.cs

```diff
@@ -75,20 +74,23 @@ public void Cleanup()
             await SendSingleMessageX509(Client.TransportType.Amqp_WebSocket_Only);
         }
 
+        [Ignore] // TODO: #171 - X509 tests are intermittently failing during CI.
         [TestMethod]
         [TestCategory("X509-Message-E2E")]
         public async Task X509_DeviceSendSingleMessage_Mqtt()
         {
             await SendSingleMessageX509(Client.TransportType.Mqtt_Tcp_Only);
         }
 
+        [Ignore] // TODO: #171 - X509 tests are intermittently failing during CI.
```

Comment:
Same as above (all other tests too).

Reply:
Thanks, Will verify and fix as necessary.